src/main/java/com/cincialert/service/IUserService.java

```diff
@@ -0,0 +1,16 @@
+package com.cincialert.service;
```

Comment:
You were also missing your interface for the user service. I added it here for development purposes